random/_types.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * A function implementing the same contract as `Math.random`, i.e. returning a
```

Comment:
We should have the term "pseudo-random number generator" somewhere in here. It's a common programming term that should make things more familiar.